examples/observing_notebooks/AcquireAndTakeSequence.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<p>I think a more detailed intro would be useful. Just to say what the script does etc and any pre-requisites (must be on-sky, fully enabled etc).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>